semantic-ui-demo/src/main/scala/react/semanticui/demo/elements/IconsComponent.scala

```diff
@@ -27,7 +27,7 @@ object IconsComponent {
   val IconAdd                        = Icon(name = "add")
   val IconCircleThin                 = Icon(name = "circle thin")
   val IconUser                       = Icon(name = "user")
-  val AllSizes: List[SemanticSize]   = List(Mini, Tiny, Small, Medium, Large, Big, Huge, Massive)
```

Comment:
Why no `Medium`?

Reply:
It throws an error:
`Warning: Failed prop type: Invalid prop ``size`` of value `medium` supplied to `Icon2`, expected one of ["mini","tiny","small","large","big","huge","massive"].
`